engine/dlib/src/remotery/lib/Remotery.c

```diff
@@ -66,10 +66,8 @@
 
 // DEFOLD
 #include <dmsdk/dlib/log.h>
-#include <dmsdk/dlib/hash.h>
-
 static const char* g_EmptyString = "<empty>"; // As seen in profile_remotery.cpp _rmt_HashString32()
-static rmtU32 g_EmptyHash = 0;
+static rmtU32 g_EmptyHash = 0x89abcdef; // Unlikely anything else would collide with this
```

Comment:
Could technically call `_rmt_HashString32()` on the string, but this avoids doing it at global scope.